inference-engine/src/vpu/graph_transformer/src/middleend/passes/adjust_data_layout.cpp

```diff
@@ -359,9 +359,16 @@ Data PassImpl::addConvertedData(
         const Model& model,
         const Data& orig,
         const StridesRequirement& reqs) {
-    auto data = model->duplicateData(
-        orig,
-        "@adjust-strides");
+    auto data = orig;
+
+    if (orig->usage() == DataUsage::Const) {
+        auto newData = model->addNewData(orig->name(), orig->desc());
```

Comment:
Can we just return newData. I mean, do not `data = model->duplicateData(...)` but `data = newData`

Reply:
fixed